plane/src/database/backend_key.rs

```diff
@@ -1,12 +1,17 @@
 use crate::{
     heartbeat_consts::{ASSUME_LOST_SECONDS, UNHEALTHY_SECONDS},
     names::BackendName,
-    types::{BackendKeyId, ClusterName, KeyConfig},
+    types::{ClusterName, KeyConfig},
 };
 use chrono::{DateTime, Utc};
 use sqlx::{postgres::types::PgInterval, PgPool};
 use std::time::Duration;
 
+pub const KEY_LEASE_RENEW_AFTER: Duration = Duration::from_secs(30);
+pub const KEY_LEASE_SOFT_TERMINATE_AFTER: Duration = Duration::from_secs(40);
+pub const KEY_LEASE_HARD_TERMINATE_AFTER: Duration = Duration::from_secs(50);
+pub const KEY_LEASE_EXPIRATION: Duration = Duration::from_secs(60);
```

Comment:
Elsewhere we have [tests that enforce invariants](https://github.com/drifting-in-space/Plane/blob/c61e5708fa925430bb15b056d0142fc7f4112f9c/plane/src/heartbeat_consts.rs#L1-L13) about durations like these. I wonder if it would be useful to have something like that here since the renewal loop logic breaks if any of these values isn't greater than the previous one.

Reply:
Good idea, done. Also it turns out `cargo nextest` does not run doctests, so I've added a command to the workflow to run these.